File: Selenium/Activities/Activity10.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver import ActionChains


with webdriver.Firefox() as driver:
    actions = ActionChains(driver)
    driver.get("https://training-support.net/webelements/drag-drop")

    print("Title of the page: ", driver.title)

    ##Findelements
    FootBall = driver.find_element(By.ID, "ball")
    DropZone1 = driver.find_element(By.ID, "dropzone1")
    DropZone2 = driver.find_element(By.ID, "dropzone2")

    actions.click_and_hold(FootBall).move_to_element(DropZone1).pause(5).release().perform()
    if((DropZone1.find_element(By.CLASS_NAME, "dropzone-text")).text == "Dropped!"):
        print("Ball was dropped in dropzone 1")

    actions.drag_and_drop(FootBall, DropZone2).pause(5).perform()
    if((DropZone2.find_element(By.CLASS_NAME, "dropzone-text")).text == "Dropped!"):
        print("Ball was dropped in dropzone 2")
    
    driver.quit()

File: Selenium/Activities/Activity10.java
package Activity;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity10 {
	
	public static void main(String[] args) {
		
		WebDriver driver = new FirefoxDriver();
		Actions builder = new Actions(driver);
		
		driver.get("https://training-support.net/webelements/drag-drop");
		System.out.println("Title of the page is: " + driver.getTitle());
		
		WebElement FootBall = driver.findElement(By.id("ball"));
		WebElement DropZone1 = driver.findElement(By.id("dropzone1"));
		WebElement DropZone2 = driver.findElement(By.id("dropzone2"));
		
		builder.clickAndHold(FootBall).moveToElement(DropZone1).pause(5000).release().build().perform();
		if(DropZone1.findElement(By.className("dropzone-text")).getText().equals("Dropped!")) {
			System.out.println("Ball was dropped in dropzone 1");
		}
		
		builder.dragAndDrop(FootBall, DropZone2).pause(5000).build().perform();
		if(DropZone2.findElement(By.className("dropzone-text")).getText().equals("Dropped!")) {
			System.out.println("Ball was dropped in dropzone 2");
		}
		driver.quit();
				
	}

}
